.github/workflows/verify-k8s.yml

```diff
@@ -170,10 +170,17 @@ jobs:
             name: fulcio-config
             namespace: fulcio-dev
           data:
+            server.yaml: |-
+              host: 0.0.0.0
+              port: 5555
+              ca: googleca
```

Comment:
We should change this to `ephemeralca` and start removing the `sed` below?  In fact I guess this will fail.

Reply:
ah right